WordPress/src/main/java/org/wordpress/android/ui/posts/PostActionHandler.kt

```diff
@@ -170,12 +171,18 @@ class PostActionHandler(
     }
 
     private fun editPostButtonAction(site: SiteModel, post: PostModel) {
-        // first of all, check whether this post is in Conflicted state.
+        // first of all, check whether this post is in Conflicted state with a more recent remote version
         if (doesPostHaveUnhandledConflict.invoke(post)) {
             postListDialogHelper.showConflictedPostResolutionDialog(post)
             return
         }
 
+        // Then check if it's in conflicted state with a remote auto-save
```

Comment:
Nitpick: 
When I see "Conflict with remote autosave" it evokes to me, that our local autosave object is in conflict with the remote autosave object. But If I'm not wrong it's not the case. This condition just checks whether an autosave object on this post exists - in other words if the post has unpublished revision.
The main reason why I think it'd be nice to distinguish these two cases explicitly is that when we introduce real autosave object conflict handling (in v2) it might become confusing.
Wdyt?